mocha-client-tests/addon/tests/lib/test.array.js

```diff
@@ -0,0 +1,8 @@
+describe('Array', function() {
+    describe('#indexOf()', function() {
+        it('should return -1 when the value is not present', function() {
+            [1,2,3].indexOf(5).should.equal(-1);
```

Comment:
This fails with `should is undefined`; it needs to use `expect()` instead, right?

![screenshot 2016-11-07 10 59 25](https://cloud.githubusercontent.com/assets/55398/20067094/72b9baee-a4d9-11e6-8d32-36b2e7104306.png)